tests/api.test.js

```diff
@@ -2,123 +2,108 @@ import { describe } from "node:test";
 import { test, expect, request } from "vitest";
 import request from "supertest";
 
-const URL = "https://app.swaggerhub.com/apis/GREENE8616/BeKindy/1.0.0/";
+const URL = "http://localhost:5000";
 
-describe("USERS table tests", () => {
-  test("GET from /api/v1/users", async () => {
-    const res = await request(URL).get("/api/v1/users");
+describe("USERS SUITE", () => {
+  test("should get users and return status code 200", async () => {
+    const res = await request(URL).get("/api/v1/users/");
     expect(res.statusCode).toBe(200);
   });
+});
 
-  test("POST to /api/v1/users", async () => {
-    const res = await request(URL)
-      .post("/api/v1/users")
-      .send({ username: "JohnDoe01", email: "JohnDoe@example.mail" });
-    expect(res.statusCode).toBe(201);
-    expect(res.body.username).toBe("JohnDoe01");
-    expect(res.body.email).toBe("JohnDoe@example.mail");
+describe("CHALLENGES SUITE", () => {
+  test("should get today's challenge and return status code 200", async () => {
+    const res = await request(URL).get("/api/v1/challenges/today");
+    expect(res.statusCode).toBe(200);
   });
 
-  test("PUT to /api/v1/users/1", async () => {
-    const res = await request(URL)
-      .put("/api/v1/users/1")
-      .send({ name: "Jane Doe", age: 30 });
-    expect(res.statusCode).toBe(200);
-    expect(res.body.name).toBe("JaneDoe");
-    expect(res.body.email).toBe("something@example.mail");
+  test("should try to get challenge with incorrect date and return status code 404", async () => {
+    const res = await request(URL).get("/api/v1/challenges/2021-01-01");
+    expect(res.statusCode).toBe(404);
```

Comment:
With the code that we have I think this request would actually return a 400 status code.

Reply:
ah i see. should I have the test check for the 400 code or change it so that it should expect a 404 code?

Reply:
I think it would be ok to just get rid of the test because the test with foobar tests for 400 and the test with 1000 tests for 404